Transformer/UI/macOS/main.swift

```diff
@@ -189,3 +190,7 @@ extension SwiftApplicationDelegate {
 let delegate: SwiftApplicationDelegate = SwiftApplicationDelegate()
 NSApplication.shared.delegate = delegate
 _ = NSApplicationMain(CommandLine.argc, CommandLine.unsafeArgv)
+
+#else
+fatalError("This demo requires AppKit on macOS to run.")
```

Comment:
```suggestion
#error("This demo requires AppKit on macOS to run.")
```

Just a silly idea. :-)

Reply:
I'd intentionally wrapped this code to prevent a compile-time error on non-macOS platforms so that someone doing a `swift build` won't have it fail when it hits this file and tries to import AppKit. For example, CI will fail on presubmit if there's a compile error on non-macOS platforms here. Using a run-time error when you attempt to use this project on a non-macOS platform isn't great, but it was the best approach I could think of at the time.

Reply:
Ah yes. That makes sense. Please ignore me. :-)